### ISSUE TITLE
Use MVC for this or something else?

### ISSUE THREAD

**Issue by nshaheed** [nshaheed/eecs448_semesterProj#1]
I think the MVC approach worked fairly well for the last project, how does everyone else feel about this?

**Comment by RobertWinslow** [comment]
I agree. The next thing I do will be to get a basic window up and to render a sprite. Then I will test to see whether it's feasible to put the render method in the sprite, and if it is, will report back on whether I think that will be more straightforward.

**Comment by RobertWinslow** [comment]
I agree. Tonight, I am working on a basic "Display one sprite" program. I will test to see how feasible per-sprite render functions are.
